packages/browserify/package.json

```diff
@@ -57,7 +57,7 @@
     "files": [
       "test/*.spec.js"
     ],
-    "timeout": "2m",
+    "timeout": "4m",
```

Comment:
another way, probably not worth doing:

1. remove the `ava` config from `browserify/package.json`
2. add a `ava.config.mjs` which reproduces everything but contains a dynamic timeout based on platform
😄